### ISSUE TITLE
The French version of Studio-Web is broken - cannot align

### ISSUE THREAD

**Issue by joanise** [ReadAlongs/Studio-Web#98]
The UI displays fine, but if I try to add data and then align it, I get this exception:

```
:5200/model/en-us/feat_params.json:1          Failed to load resource: the server responded with a status of 404 (Not Found)
core.mjs:8506 ERROR Error: Failed to fetch model/en-us/feat_params.json :Not Found
    at soundswallower.web.js:20:26816
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (asyncToGenerator.js:3:24)
    at _next (asyncToGenerator.js:22:9)
    at _ZoneDelegate.invoke (zone.js:372:26)
    at Object.onInvoke (core.mjs:24313:33)
    at _ZoneDelegate.invoke (zone.js:371:52)
    at Zone.run (zone.js:134:43)
    at zone.js:1275:36
    at _ZoneDelegate.invokeTask (zone.js:406:31)
handleError @ core.mjs:8506
Show 1 more frame
```

![image](https://user-images.githubusercontent.com/33264708/218571067-4b85831c-4f31-4dad-ad57-fa7a5e8e4a14.png)

The trace was different but the error similar when I tried again after a reload:

```
soundswallower.service.ts:91 ERROR Error: Failed to fetch model/en-us/feat_params.json :Not Found
    at soundswallower.web.js:20:26816
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (asyncToGenerator.js:3:24)
    at _next (asyncToGenerator.js:22:9)
    at _ZoneDelegate.invoke (zone.js:372:26)
    at Object.onInvoke (core.mjs:24313:33)
    at _ZoneDelegate.invoke (zone.js:371:52)
    at Zone.run (zone.js:134:43)
    at zone.js:1275:36
    at _ZoneDelegate.invokeTask (zone.js:406:31)
```

![image](https://user-images.githubusercontent.com/33264708/218571959-7eb30e5d-5dac-4abe-8fac-732e4fbde6bf.png)

**Comment by joanise** [comment]
@dhdaines Can you look at this?

**Comment by joanise** [comment]
To reproduce, run:

    nx start-fr studio-web
    nx serve-web-api studio-web

and try to align using http://localhost:5200/

**Comment by dhdaines** [comment]
Sure, looks like something broke in the build with the change to nx, it isn't copying the assets correctly into both languages...

**Comment by dhdaines** [comment]
Seems to be fixed in #111 (though deployment is still broken...)
